tests/foreman/ui/test_discoveredhost.py

```diff
@@ -143,6 +145,183 @@ def test_positive_pxe_less_discovery(self):
                 self._assertdiscoveredhost(hostname)
                 self.assertIsNotNone(self.discoveredhosts.search(hostname))
 
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_pxe_less_with_dhcp_semiauto(self):
```

Comment:
As I understand that this may need mouse clicks during the provisioning stage to select few options. Just curious, Is it possible to automate this?